ui/src/app/cohort-search/cohort-search/cohort-search.component.tsx

```diff
@@ -104,7 +104,51 @@ interface State {
   treeSearchTerms: string;
 }
 
+const css = `
```

Comment:
May need to tweak/add some styles here to match the mock, can do in the follow-up PR if you prefer. Looks like mainly the check icon is too big and increases the the total height of the growl

Reply:
I'll take care of this in follow-up PR for criteria pop up